experimental/behave/src/main/kotlin/net/corda/behave/network/Network.kt

```
@@ -242,10 +253,10 @@ class Network private constructor(
         isDoormanNMSRunning = true
     }
 
-    private fun runCommand(command: JarCommand, noWait: Boolean = false) {
+    private fun runCommand(command: JarCommand, noWait: Boolean = false):Command? {
         if (!command.jarFile.exists()) {
             log.warn("Jar file does not exist: ${command.jarFile}")
-            return
+            return null
```

Can we just throw an exception here? Instead of returning a nullable `Command`?